.github/workflows/aws-deployment.yml

```diff
@@ -0,0 +1,95 @@
+---
+    name: AWS Deployment
+
+    on:
+      workflow_dispatch:
+        inputs:
+            environment:
+                description: "Environment to deploy to"
+                default: "staging"
+                options:
+                    - staging
+                    - production
+                required: true
+                type: choice
+
+    env:
+      AWS_ACCOUNT_ID: ${{ vars.AWS_PUBLIC_DATA_RELEASES_ACCOUNT_ID }}
```

Comment:
Double-checking - are these variables set?

Reply:
Looks like these would need to go in Repository Variables, here: https://github.com/AllenInstitute/aics-fms-file-explorer-app/settings/variables/actions

Reply:
Looks good to me